backend/src/main/java/fk/prof/backend/http/handler/RecordedProfileRequestHandler.java

```diff
@@ -27,7 +39,14 @@ public RecordedProfileRequestHandler(RoutingContext context, CompositeByteBufInp
 
   @Override
   public void handle(Buffer requestBuffer) {
-//    System.err.println(String.format("buffer=%d, chunk=%d", runningBuffer.length(), requestBuffer.length()));
+    //    try { logger.debug(String.format("buffer=%d, chunk=%d", inputStream.available(), requestBuffer.length())); } catch (Exception ex) {}
+    histChunkSize.update(requestBuffer.length());
```

Comment:
Had a discussion with @janmejay about this. He suggested to put byte hist and idle timer hist for chunked call (/profile)